### ISSUE TITLE
Too Many Request 429 Throttling Error

### ISSUE THREAD

**Issue by jayeshmanani** [sergioteula/python-amazon-paapi#35]

**Comment by b0lle** [comment]
can you provide your code, response, version? When do thus error occurs? Do you use a scheduler? Any details would be great.

**Comment by migcm** [comment]
Related with #32.

**Comment by jayeshmanani** [comment]
No, I don't use the scheduler or something, I have just used the same code as provided by you, just changed the key of the API and secret only, But, every time I got the same, I used the limited request using this code given below, 
In-country I have also tried the US but not working properly, I got the throttling error.

Key = ' My key '
secret = ' secret key '
tag = ' my tag '
country = 'IN' 

from amazon.paapi import AmazonAPI
amazon = AmazonAPI(KEY, SECRET, TAG, COUNTRY)
product = amazon.get_product('B01N5IB20Q')
print(product.title)

**Comment by faddison** [comment]
Do you get the same error using scratchpad? If so, then its your key/account, not the code.

https://webservices.amazon.com/paapi5/scratchpad/index.html
